syntax = "proto3";

package pb.core;


enum OperateProtocol {
    OP_None = 0;

    //业务操作码，从1000开始
    OP_AccountLoginRequest = 1001;
    OP_AccountLoginResponse = 1002;

    OP_ClientLoginRequest = 1101;
    OP_ClientLoginResponse = 1102;

    OP_EchoRequest = 1103;
    OP_EchoResponse = 1104;

    OP_AsyncEchoRequest = 1105;
    OP_AsyncEchoResponse = 1106;


    OP_HandshakeInitRequest = 1205;
    OP_HandshakeInitResponse = 1206;
    OP_HandshakeEstablishedRequest = 1207;
    OP_HandshakeEstablishedResponse = 1208;
}
